crates/core_arch/src/x86/sse41.rs

```diff
@@ -1148,6 +1142,35 @@ pub unsafe fn _mm_test_mix_ones_zeros(a: __m128i, mask: __m128i) -> i32 {
     _mm_testnzc_si128(a, mask)
 }
 
+/// Load 128-bits of integer data from memory into dstt. mem_addr must be aligned on a 16-byte
+/// boundary or a general-protection exception may be generated. To minimize caching, the data
+/// is flagged as non-temporal (unlikely to be used again soon)
+///
+/// [Intel's documentation](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_stream_load_si128)
+///
+/// # Safety of non-temporal stores
+///
+/// After using this intrinsic, but before any other access to the memory that this intrinsic
+/// mutates, a call to [`_mm_sfence`] must be performed by the thread that used the intrinsic. In
+/// particular, functions that call this intrinsic should generally call `_mm_sfence` before they
+/// return.
+///
+/// See [`_mm_sfence`] for details.
+#[inline]
+#[target_feature(enable = "sse4.1")]
+#[cfg_attr(test, assert_instr(movntdqa))]
+#[stable(feature = "simd_x86", since = "1.27.0")]
+pub unsafe fn _mm_stream_load_si128(mem_addr: *const __m128i) -> __m128i {
```

Comment:
Could you split out the addition of new stable intrinsics under an existing feature flag into a new PR, so that they can be FCPed if necessary?

Reply:
There are 5-6 missed functions in sse2, sse3, ssse3 and avx2. What feature flag should I use?

Reply:
Oh, I have no objection to you proposing they be insta-stabilized under that feature flag, it's just better to have insta-stable additions be in their own PR.

Reply:
Oh, ok. So they should be insta-stabilized under `simd_x86`? and what should be the since argument be? (I don't know enough about stabilization)

Reply:
The `since` argument should be either the same version as the rest, or the current nightly, I think? I'm not entirely sure either. This is an edge case of a "should've been in the original intrinsic set" thing.

Yes, T-libs-api will look at it and discuss whether or not to simply merge it or whether to FCP it as a formality or what. In general, adding or stabilizing new public API should be split from changing implementation of existing public API, unless someone says something like "let's change this one thing and then stabilize it".

Reply:
Ok, I will send a PR for these missing intrinsics. We cannot use simd_x86 feature with any version other than 1.27.0 (it doesn't compile)

Reply:
Also can we do anything about `_mm_malloc` and `_mm_free`?

Reply:
I don't think we should try that here.